src/chi/components/modal/modal.scss

```diff
@@ -11,7 +11,7 @@
     background-color: set-color(white);
     border-radius: $border-radius;
     box-shadow: 0 0 25px 10px rgba(set-color(black), 0.15);
-    margin: 128px auto 0;
+    margin: auto 0;
```

Comment:
Previous value was setting left and right to auto. They're now set to 0. Shouldn't this just be `margin: auto;` ?

Reply:
done